src/engraving/types/symnames.cpp

```diff
@@ -3453,9 +3453,9 @@ constexpr const std::array<const char*, size_t(SymId::lastSym) + 1> SymNames::s_
     QT_TRANSLATE_NOOP("engraving/sym", "Double sharp raised by three syntonic commas"),
     QT_TRANSLATE_NOOP("engraving/sym", "Double sharp lowered by two syntonic commas"),
     QT_TRANSLATE_NOOP("engraving/sym", "Double sharp raised by two syntonic commas"),
-    "Enharmonically reinterpret accidental almost equal to",
-    "Enharmonically reinterpret accidental equals",
-    "Enharmonically reinterpret accidental tilde",
+    QT_TRANSLATE_NOOP("engraving/sym", "Enharmonically reinterpret accidental tilde"),
+    QT_TRANSLATE_NOOP("engraving/sym", "Enharmonically reinterpret accidental almost equal to"),
+    QT_TRANSLATE_NOOP("engraving/sym", "Enharmonically reinterpret accidental equals"),
```

Comment:
I believe the order of these must match that in `symid_p.h`

Reply:
Doesn't it?

Reply:
You're of course right